Add header with sapling name and button to propose circuits

eloaverona opened the pull request:
### To test

1. Run 
```
docker-compose -f docker/docker-compose.yaml  build sapling-dev-server-alpha sapling-dev-server-beta 
```
2. Run 
```
docker-compose -f docker/docker-compose.yaml up
```

3. Go to `localhost:3030/circuits`:
<img width="1273" alt="Screen Shot 2020-04-09 at 10 14 47 AM" src="https://user-images.githubusercontent.com/14094978/78910744-fed8a180-7a4a-11ea-8ddd-1a36aed6422d.png">

4. The Propose Circuit button collapses if the screen width is small:
<img width="1273" alt="Screen Shot 2020-04-09 at 10 15 43 AM" src="https://user-images.githubusercontent.com/14094978/78910883-24fe4180-7a4b-11ea-90c5-ca9473851524.png">
 

P.S The button currently does not work.

Commit:
Add header component with sapling name

Signed-off-by: Eloá Franca Verona <eloafran@bitwise.io>

Commit:
Add propose circuit button

The button does not currently work.

Signed-off-by: Eloá Franca Verona <eloafran@bitwise.io>

dplumb94 commented on saplings/circuits/src/components/MainHeader.scss:
```diff
@@ -0,0 +1,29 @@
+ * limitations under the License.
+ */
+
+ .main-header {
```
make sure the scss code uses 2 space tabs and starts at the margin

eloaverona replied:
updated

dplumb94 commented on saplings/circuits/src/components/ProposeCircuitButton.scss:
```diff
@@ -0,0 +1,49 @@
+ * limitations under the License.
+ */
+
+.propose-circuit-btn {
```
Fix tab size

eloaverona replied:
updated

dplumb94 commented on saplings/circuits/src/components/ProposeCircuitButton.scss:
```diff
@@ -0,0 +1,49 @@
+    justify-content: space-around;
+    height: 3rem;
+    width: 15rem;
+
```
add `cursor: pointer` for buttons

eloaverona replied:
updated

Commit:
Fix splinter-ui image name in docker-compose

The images for the splinter-ui alpha and beta nodes 
should not use the same name. It causes the image
used by the alpha node to be overwritten with
the config for the beta node. 

Signed-off-by: Eloá Franca Verona <eloafran@bitwise.io>

dplumb94 commented on saplings/circuits/src/components/ProposeCircuitButton.scss:
```diff
@@ -0,0 +1,50 @@
+    width: auto;
+
+    // hide button text in small screens
+    .btn-text{
```
Nitpick: add space before brace (I'm looking in to how to lint the scss)

agunde406 reviewed (changes requested):
If you try to go to http://localhost:3030/circuits without registering/logging in I got this view:


<img width="1440" alt="Screen Shot 2020-04-14 at 7 30 49 AM" src="https://user-images.githubusercontent.com/18215486/79225163-f0f79780-7e21-11ea-9ea6-67b33ff6e3e2.png"> 

The url for this view is http://localhost:3030/login so it is trying to redirect. This was on firefox

agunde406 reviewed (approved):
Observed issue is a problem with canpy js and the login sapling. A bug has been created.
